<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="VerticalScaleWidget" name="yScale"/>
   </item>
   <item row="0" column="1">
    <widget class="PlotSceneWidget" name="graphicsPlot"/>
   </item>
   <item row="1" column="1">
    <widget class="HorizontalScaleWidget" name="xScale"/>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lower Bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teLower">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Upper Bound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teUpper">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNow">
       <property name="text">
        <string>Now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbAddSecs">
       <property name="suffix">
        <string>sec</string>
       </property>
       <property name="prefix">
        <string>+</string>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAutoscale">
       <property name="text">
        <string>Autoscale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbShowDate">
       <property name="text">
        <string>Show Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>N. of Points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBuffer">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPeriod">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotSceneWidget</class>
   <extends>QGraphicsView</extends>
   <header>plotscenewidget.h</header>
  </customwidget>
  <customwidget>
   <class>HorizontalScaleWidget</class>
   <extends>QWidget</extends>
   <header>horizontalscalewidget.h</header>
  </customwidget>
  <customwidget>
   <class>VerticalScaleWidget</class>
   <extends>QWidget</extends>
   <header>verticalscalewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
